patches/net-BUILD.gn.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/net/BUILD.gn b/net/BUILD.gn
+index 9d59189d80aea8f99b73020f36a578a96667af37..d5aa98b8f2f44f6acec58c124c262bd1ddcabe31 100644
+--- a/net/BUILD.gn
++++ b/net/BUILD.gn
+@@ -355,6 +355,9 @@ component("net") {
+     "ssl/token_binding.h",
+     "third_party/quic/core/quic_error_codes.cc",
+     "third_party/quic/core/quic_error_codes.h",
++    "//brave/chromium_src/net/base/host_port_pair_auth.cc",
++    "//brave/chromium_src/net/base/url_auth_util.cc",
```

Comment:
same thing applies to these

Reply:
614d17f6149feded4eb20f9049a9050ba1a5cfd2
a9a045e0dd404ef76cc00a727f41118cc4ecc17d